config/prod/prow/k8s-testgrid/k8s-testgrid.yaml

```diff
@@ -60,11 +60,13 @@ dashboards:
 - name: knative-0.19
 - name: knative-0.20
 - name: knative-0.21
+- name: knative-0.22
 - name: knative-gcp
 - name: knative-sandbox-0.18
```

Comment:
Should this have been removed?

Reply:
It would not as long as there is still a single repo under knative-sandbox haven't cut 0.22 yet